client/components/domains/domain-suggestion-flag/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' );
+
+/**
+ * Internal dependencies
+ */
+var Flag = require( 'components/flag' );
+
+const DomainSuggestionFlag = React.createClass( {
+	propTypes: {
+		domain: React.PropTypes.string
+	},
+
+	render() {
+		var newTLDs = ['.live'];
+		var i;
```

Comment:
This should a single `var` declaration as per our [guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#declaring-variables-with-var) (OMG, sorry for being so picky...)